docs/PinnedImageSets.md

```diff
@@ -0,0 +1,42 @@
+# Updating a cluster in a disconnected environment without a local registry
+
+You can use `PinnedImageSet` custom resources (CRs) to pin and pre-load release images to a defined machine config pool, which enables you to update an OpenShift Container Platform (OCP) cluster in a disconnected environment without needing an image registry in the environment.
+This can be useful for clusters that were installed using the [OpenShift-based Appliance builder](https://access.redhat.com/articles/7065136), which might have been deployed in disconnected environments where building a local image registry is not practical.
+
+To update a cluster in a connected environment, or in a disconnected environment with a registry, see the product documentation for [updating a cluster](https://docs.openshift.com/container-platform/4.16/updating/understanding_updates/intro-to-updates.html).
+
+**Warning**: The `PinnedImageSet` CR is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+For more information about the support scope of Red Hat Technology Preview features, see [Technology Preview Features Support Scope](https://access.redhat.com/support/offerings/techpreview/).
+
+**Important**: Known issues have been observed with this feature, causing cluster updates to fail without a registry present.
+
+## Prerequisites
+
+* You have access to your cluster with administrator privileges.
+
+## Procedure
+
+1. Go to the [OpenShift Container Platform release repository](https://quay.io/repository/openshift-release-dev/ocp-release) and retrieve the release images for your target update version.
```

Comment:
Should we add how to retrieve the list of release images?
```
$ oc adm release info quay.io/openshift-release-dev/ocp-release:4.17.8-x86_64 --output=json | jq "[.references.spec.tags[] | .from.name]"
+
$ oc adm release info quay.io/openshift-release-dev/ocp-release:4.17.8-x86_64 | awk '/Pull From/ { print "    - name: "$3 }'
```

Reply:
I can add these commands to this step, sure. Could you provide a short description of what each command is doing? I'll need it so that I can write my steps like "Take action X by running the following command:" (so it doesn't need to be super descriptive)

Reply:
In the PinnedImageSet we should add the release images (1st command) + the base image of the release itself (2nd command).

Retrieve the list of OCI images used in the release with:
```
$ oc adm release info quay.io/openshift-release-dev/ocp-release:<version>-<arch> --output=json | jq "[.references.spec.tags[] | .from.name]"
```

And you should add also the reference to the release image, as it will be requested during the upgrade process. The easiest way to extract this data from the release is by using the command:
```
$ oc adm release info quay.io/openshift-release-dev/ocp-release:<version>-<arch> | awk '/Pull From/ { print " - name: "$3 }'
```